CHANGELOG.md

```diff
@@ -2,12 +2,24 @@
 
 **See [the release process docs](docs/howtos/cut-a-new-release.md) for the steps to take when cutting a new release.**
 
-[Full Changelog](https://github.com/mozilla/application-services/compare/v0.15.0...master)
+[Full Changelog](https://github.com/mozilla/application-services/compare/v0.16.0...master)
 
-## Places
+
+# 0.16.0 (_2019-02-06_)
+
+[Full Changelog](https://github.com/mozilla/application-services/compare/v0.15.0...v0.16.0)
+
+## General
 
 ### What's New
 
+- iOS builds now target v11.0. ([#614](https://github.com/mozilla/application-services/pull/614))
+- Preparatory infrastructure for megazording iOS builds has landed.([#625](https://github.com/mozilla/application-services/pull/625))
+
+## Places
+
+### Breaking Changes
+
 - Several new methods on PlacesConnection (Breaking changes for classes implementing PlacesAPI):
```

Comment:
Are these "breaking changes" in the "your existing code in another repo will stop working", or is this just flagging that work as to be done to implement the new methods?  It's not clear from the details below what breakage will be caused by the new methods.

Reply:
It will be a compilation failure. You're not allowed to implement only some of the methods in an interface, and so us adding new methods means that anybody using this interface must update or their code will break. I wouldn't refer to the other as a breaking change.